### DISCUSSION TITLE
Wize Cam OG (Model WYZECGS-RB) supported?

### DISCUSSION THREAD

**Discussion by dashxdr** [openmiko/openmiko#126] [Q&A]
Home Depot has a sale on these for under $17, are they supported?
Wyze Cam OG
Color Night Vision
Indoor/Outdoor
Wired
There is a 5V Usb adapter included.
Model: WYZECGS-RB

Thanks!

**Comment by martinbutt** [comment]
It depends on which chip it uses. Openmiko supports the Ingenic T20. Newer Wyze cams use the T31. I can't find any technical documents or teardowns to confirm which chip the OG uses, but it's likely not supported. You can always try it and return it.

**Reply by dashxdr** [reply]
After a destructive "teardown" I've exposed the interior circuitry. I see a big square chip RTL8735BM, and another 8 pin surface mount chip that looks like either a flash or ram device, I can read cFeon but the ID numbers are very small. Let me know how I can assist in supporting this device.
Regarding the teardown, it's pretty clear glue is involved so non-destructive disassembly is probably unlikely. I can provide photos if they'd be of use.

**Reply by martinbutt** [reply]
The RTL8735B is an integrated processor for cameras, so it sounds like Wyze have dropped the Ingenic chips in favor of those.

**Reply by dashxdr** [reply]
Ok in case this is of use to anyone else, I could have disassembled the unit without damaging it. There is a glued on thin face plate on the camera side. This can be pried off with a thin screwdriver or guitar pick. This will expose 3 of the 4 corners that have rubbery white material jammed in. With a pin it would be possible to pry those out, and inside are plastic tubes with a small phillips screw at the bottom. Unscrew these 3 screws and I bet the guts can be slid out. Behind the guts is the sealed conduit to the power cable, and the speaker assembly which is screwed to the rear of the white plastic case, and wires connect to the electronics.

**Reply by dashxdr** [reply]
Here's a picture
![guts](https://github.com/openmiko/openmiko/assets/2752262/59dbfcc5-12ba-4715-9be9-24ea5633c210)

**Reply by dashxdr** [reply]
Lots of discussion/progress over here: https://github.com/gtxaspec/wz_mini_hacks/discussions/638

**Reply by dashxdr** [reply]
And I'm trying to get technical support from amebaiot/realtek here: https://forum.amebaiot.com/t/rtl8735bm-programming-not-working-with-uartfwburn-linux-ping-failure/2483
